trie/trieStorageManager.go

```diff
@@ -419,12 +419,12 @@ func (tsm *trieStorageManager) SetCheckpoint(
 	case tsm.checkpointReq <- checkpointEntry:
 	case <-tsm.closer.ChanClose():
 		tsm.ExitPruningBufferingMode()
-		tsm.safelyCloseChan(leavesChan)
+		safelyCloseChan(leavesChan)
 		stats.SnapshotFinished()
 	}
 }
 
-func (tsm *trieStorageManager) safelyCloseChan(ch chan core.KeyValueHolder) {
+func safelyCloseChan(ch chan core.KeyValueHolder) {
```

Comment:
why did we remove this method from trieStorageManager ?

Reply:
This func does not need anything from trieStorageManager, so it should not be a methd. If it is a normal func, it can be called from other implementations of StorageManager.

Reply:
:+1: